lib/happner-client.js

```diff
@@ -56,9 +56,9 @@
     this.__implementors = new ImplementorsProvider(this, this.__connection);
     this.__operations = new OperationsProvider(this, this.__connection, this.__implementors);
     this.__requestTimeout =
-      typeof opts.requestTimeout === 'number' ? opts.requestTimeout : 10 * 1000;
+      typeof opts.requestTimeout === 'number' ? opts.requestTimeout : 120 * 1000;
```

Comment:
would it perhaps make sense to keep the requestTimeout at half the responseTimeout, as before?